service/src/test/java/bio/terra/cbas/pact/EcmPactTest.java

```diff
@@ -0,0 +1,81 @@
+package bio.terra.cbas.pact;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.PactSpecVersion;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import bio.terra.cbas.config.EcmServerConfiguration;
+import bio.terra.cbas.dependencies.ecm.EcmClient;
+import bio.terra.cbas.dependencies.ecm.EcmService;
+import bio.terra.common.iam.BearerToken;
+// import bio.terra.externalcreds.pact.ProviderStates;
```

Comment:
Once the ECM provider verification has been merged ([PR](https://github.com/DataBiosphere/terra-external-credentials-manager/pull/189)) I'll want to uncomment this and replace the plain string states below with the corresponding enums.